.github/scripts/deploy-fleet.sh

```diff
@@ -39,7 +39,7 @@ eventually helm upgrade --install fleet charts/fleet \
 # wait for controller and agent rollout
 kubectl -n cattle-fleet-system rollout status deploy/fleet-controller
 { grep -E -q -m 1 "fleet-agent-local.*1/1"; kill $!; } < <(kubectl get bundles -n fleet-local -w)
-kubectl -n cattle-fleet-system rollout status deploy/fleet-agent
+kubectl -n cattle-fleet-system rollout status statefulset/fleet-agent
```

Comment:
* .github/scripts/deploy-fleet-latest-release.sh - not possible until release exists that uses a statefulset, because that script deploys the latest released fleet

* .github/scripts/setup-latest-rancher.sh - are you saying this should also wait for agents?
    .github/scripts/setup-rancher.sh - add another wait here?
    .github/workflows/rancher-upgrade-fleet.sh - I think this waits for agent bundles, which is a stronger check than waiting for the statefulset

Reply:
Oh of course, this is an agent-only change, so my comment does not make much sense (except perhaps for `deploy-fleet-latest-release.sh`, which as you've noticed needs to wait until Fleet is released with stateful sets).
Sorry for the noise.